README.md

```diff
@@ -81,30 +81,30 @@ You can call any method on authenticated user. For example, let's send message t
   var result = await client.GetContactsAsync();
 
   //find recipient in contacts
-  var user = result.Users.lists
+  var user = result.Users
```

Comment:
did you miss to write .Lists here?

Reply:
no, there is no need for it and having it cause compile error

Reply:
ok good to know